Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -978,7 +979,7 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
         fixIts: [
           FixIt(
             message: ReplaceTokensFixIt(replaceTokens: [.binaryOperator("==")], replacements: [node.equal]),
-            changes: [.makeMissing(unexpected), .makePresent(node.equal, leadingTrivia: [])]
```

Comment:
I found two places where we don't need to add trivia

Reply:
Nice. I suspect there might be a few more as well. Haven’t checked though.

Reply:
Tried to do a quick look, didn't find any